packages/blockchain/src/airdrop/createThirdwebAirdropContract.ts

```diff
@@ -16,21 +16,15 @@ export async function createThirdwebAirdropContract({
   chainId,
   adminPrivateKey,
   tokenAddress,
-  proxyFactoryAddress,
-  implementationAddress,
   expirationTimestamp,
-  nullAddressAmount,
-  tokenDecimals
+  nullAddressAmount
 }: {
   recipients: Recipient[];
   chainId: number;
   adminPrivateKey: `0x${string}`;
   tokenAddress: Address;
-  proxyFactoryAddress: Address;
-  tokenDecimals: number;
-  implementationAddress: Address;
   expirationTimestamp: bigint;
-  nullAddressAmount?: number;
+  nullAddressAmount?: string;
```

Comment:
i changed this method a bit - we were already passing in values as stringified wei values, as i understand. but then we were converting to floats to add the total values up and then re-formatting them. so instead i just add numbers as BigInt which should be safer.